index.html

```diff
@@ -175,8 +175,8 @@ <h1>Landing Page</h1>
     </section>
 
     <section class="division about-me">
```

Comment:
- [ ] According to the design, the desktop version has a navigation bar with links various sections of page. The hamburger buttons is removed from the navigation bar in the desktop version. Please kindly fix that as it is part of the project requirements and best practices in the industry to stick to the clients design. See images below.

***Your implementation***
![raph1](https://user-images.githubusercontent.com/65852621/134439434-aceb6b32-c343-45a2-84df-2ebf708771d8.png)

***The design***
![raphdesign](https://user-images.githubusercontent.com/65852621/134439462-7b10f3bf-1fc8-4c82-8b63-4eb07106b40d.png)